how-to/panel-login.md

```diff
@@ -0,0 +1,67 @@
+---
+id: panel-login
+title: Configure Endpoint Auto Login
+description: Allow automatic or unatteded authentication for PlaceOS Frontend Applications
+sidebar_position: 8
+---
+
+By default, PlaceOS requires users to authentication against frontend applications (Workplace, Kiosks, Concierge etc.).
+Endpoint Auto Login allows you to configure panels to authenticate against PlaceOS Automatically. 
+This removes the requirement for users to manually enter credentials to authenticate the application.
+This feature is most commonly used with unattended user interfaces, such as map and visitor kiosks.  
+
+
+## Prerequisites
+1. Confirm the final UAT and PROD URLs of the web apps
+2. Ensure that the **DNS** entries for these URLs are active and forwarding to the server(s)
+3. Ensure that the SSL certificates for the above domains are signed and recognized as secure
+
+
+## Step 1: Create a local user account
+
+1. Login as an admin to Backoffice
+2. On the **Users** click **Add new**
+3. Enter the required information, we recommend setting the username as `Touchpanel User`
```

Comment:
3. Enter the required information, including a descriptive username (e.g. `Wayfinding Displays`)

Reply:
Fixed in commit da00f5b97e164219504ce7b07f2ac6bc3cff28b8